src/selectors/prometheus/selectors.js

```diff
@@ -28,3 +28,40 @@ export const getLastUtilizationStat = response => {
   const history = getUtilizationVectorStats(response);
   return history ? history[history.length - 1] : null;
 };
+
+export const getTopConsumerVectorStats = result => {
+  let maxVal = 0;
+
+  result.forEach(namespace => {
+    namespace.values.forEach(value => {
```

Comment:
you could also use lodash `flatMap` here and have one `Math.max` call

Reply:
yes, flatMap will be a better solution.